src/config.ts

```diff
@@ -207,7 +207,8 @@ export const APIMETA: MetaDataItemsObject = {
       GARD: ['mondo.xrefs.gard', 'disease_ontology.xrefs.gard'],
       HP: ['mondo.xrefs.hp'],
       SNOMEDCT: ['mondo.xrefs.sctid', 'umls.snomed.preferred', 'umls.snomed.non-preferred'],
-      NCIT: ['mondo.xrefs.ncit', 'disease_ontology.xrefs.ncit']
+      NCIT: ['mondo.xrefs.ncit', 'disease_ontology.xrefs.ncit'],
+      MEDDRA: ['mondo.xrefs.meddra', 'disease_ontology.xrefs.meddra'],
```

Comment:
will the trailing comma here cause any problems?

Reply:
I don't think so? There's a trailing comma [here](https://github.com/biothings/biomedical_id_resolver.js/blob/e9149760a1361389b27d5b674545f897068a16a6/src/config.ts#L28) and [here](https://github.com/biothings/biomedical_id_resolver.js/blob/e9149760a1361389b27d5b674545f897068a16a6/src/config.ts#L184)....throughout the file really.

I don't know the syntax though...